libkbfs/folder_block_manager.go

```diff
@@ -944,10 +944,8 @@ func (fbm *folderBlockManager) finalizeReclamation(ctx context.Context,
 		gco.AddUnrefBlock(BlockPointer{ID: id})
 	}
 
-	// For now, pretend to be a rekey so the service suppresses
-	// popups.  TODO: add a more specific behavior type for QR.
 	ctx, err := makeExtendedIdentify(
-		ctx, keybase1.TLFIdentifyBehavior_KBFS_REKEY)
+		ctx, keybase1.TLFIdentifyBehavior_KBFS_QR)
```

Comment:
Please leave in some comment about how this asks the service to suppress popups for successful identifies.